bundles/AdminBundle/Controller/Admin/Document/EmailController.php

```diff
@@ -74,14 +74,14 @@ public function getDataByIdAction(Request $request)
         }
         Element\Editlock::lock($request->get('id'), 'document');
 
-        $email = Document\Email::getById($request->get('id'));
+        $emailFromDatabase = Document\Email::getById($request->get('id'));
 
-        if (!$email) {
+        if (!$emailFromDatabase) {
             throw $this->createNotFoundException('Email not found');
         }
 
-        $email = clone $email;
-        $email = $this->getLatestVersion($email);
+        $emailFromDatabase = clone $emailFromDatabase;
+        $email = $this->getLatestVersion($emailFromDatabase);
```

Comment:
Maybe we should move this functionality directly to `\Pimcore\Bundle\AdminBundle\Controller\Admin\Document\DocumentControllerBase::preSendDataActions()`

See my code suggestion there. 
Then I think we don't have to touch all the controllers and have it in a central place. wdyt?